camel-core/src/main/java/org/apache/camel/impl/ThrottlingExceptionRoutePolicy.java

```diff
@@ -71,11 +71,11 @@
     private ThrottlingExceptionHalfOpenHandler halfOpenHandler;
 
     // stateful information
-    private final AtomicInteger failures = new AtomicInteger();
-    private final AtomicInteger state = new AtomicInteger(STATE_CLOSED);
-    private volatile Timer halfOpenTimer;
-    private volatile long lastFailure;
-    private volatile long openedAt;
+    private Timer halfOpenTimer;
```

Comment:
Please dont change this back as those changes are for threading, you need to use volatile when using primitives